src/modules/components/proposal/proposalActions/proposalActionsContainer/proposalActionsContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import { cloneElement, isValidElement, useState, type ReactElement } from 'react';
+import { Accordion, Button, Card, Heading, type IAccordionContainerBaseProps } from '../../../../../core';
+import { useOdsModulesContext } from '../../../odsModulesProvider';
+import { type IProposalActionsActionProps } from '../proposalActionsAction';
+
+export interface IProposalActionsContainerProps extends Omit<IAccordionContainerBaseProps<true>, 'isMulti'> {
+    /**
+     * Actions as custom children to render
+     */
+    children?: ReactElement<IProposalActionsActionProps> | Array<ReactElement<IProposalActionsActionProps>>;
+    /**
+     * Additional class names for styling
+     */
+    className?: string;
+}
+
+export const ProposalActionsContainer: React.FC<IProposalActionsContainerProps> = (props) => {
+    const { className, children } = props;
+    const [expandedItems, setExpandedItems] = useState<string[]>([]);
+    const { copy } = useOdsModulesContext();
+
+    const childrenCount = Array.isArray(children) ? children.length : children ? 1 : 0;
+
+    const handleToggleAll = () => {
+        if (expandedItems.length === childrenCount) {
+            setExpandedItems([]);
+        } else {
+            setExpandedItems(Array.from({ length: childrenCount }, (_, index) => `${index}-action`));
+        }
+    };
+
+    const handleToggleItem = (index: number) => {
+        if (expandedItems.includes(`${index}-action`)) {
+            setExpandedItems(expandedItems.filter((item) => item !== `${index}-action`));
+        } else {
+            setExpandedItems([...expandedItems, `${index}-action`]);
+        }
+    };
+
+    const handleAccordionValueChange = (value: string[] | undefined) => {
+        setExpandedItems(value ?? []);
+    };
+
+    const renderChildren = () => {
+        if (!children) {
+            return null;
+        }
+
+        if (Array.isArray(children)) {
+            return children.map((child, index) => {
+                if (isValidElement(child)) {
+                    return cloneElement(child, {
+                        onToggle: () => handleToggleItem(index),
+                    });
+                }
+                return child;
+            });
+        }
+
+        if (isValidElement(children)) {
+            return cloneElement(children, {
+                onToggle: () => handleToggleItem(0),
+            });
+        }
+
+        return children;
+    };
```

Comment:
As we're setting a callback on the accordion.container, we don't probably need to pass a `onToggle` property to each child component. Radix-ui should trigger the callback set on the accordion-container when a child component open state changes